model/incoming_webhook_test.go

```diff
@@ -179,30 +143,18 @@ func TestIncomingWebhookRequestFromJson(t *testing.T) {
 
 		// After it has been decoded, the JSON string won't contain the escape char anymore
 		expected := strings.Replace(text, `\"`, `"`, -1)
-		if iwr == nil {
-			t.Fatal("IncomingWebhookRequest should not be nil")
-		}
-		if iwr.Text != expected {
-			t.Fatalf("Sample %d text should be: %s, got: %s", i, expected, iwr.Text)
-		}
+		require.NotNil(t, iwr, "IncomingWebhookRequest should not be nil")
+		require.Equalf(t, expected, iwr.Text, "Sample %d text should be: %s, got: %s", i, expected, iwr.Text)
```

Comment:
I think we can remove the error messages here as well

```suggestion
		require.Equal(t, expected, iwr.Text)
```